trace_decoder/Cargo.toml

```diff
@@ -41,14 +41,26 @@ zk_evm_common = { workspace = true }
 
 [dev-dependencies]
 alloy = { workspace = true }
+alloy-compat = "0.1.0"
```

Comment:
I see you've published your own version of the compat crate we had made, any reason you didn't publish ours directly?

Reply:
I didn't know what the overhead was in creating a new `0xPolygonZero` repo, adding CI etc 🤷 
Happy to move, though I was planning to upstream this to `alloy` at some point :)

Reply:
This is basically the compat repo we already have right? So no need for a new repo I think.
For sure if this is soon to be upstreamed then no problem, just curious :)